### ISSUE TITLE
username used to send data to ODK Aggregate

### ISSUE THREAD

**Issue by sadiqkhoja** [opendatakit/aggregate#236]
Feature Request:

As an administrator/researcher, I want to see the username (that is used to connect ODK Aggregate Server not Form Metadata) of the data collector in submission page and exported csv file so that I can identify and authenticate data collector.

I have seen this post https://forum.opendatakit.org/t/best-practices-to-identify-the-interviewers-using-odk-collect-auth-barcode/3896 and solution suggested by @yanokwa doesn't fully meet our requirement because user metadata phonenumber, username and email can be change by user himself, and we don't want anyone to impersonate other user.

**Comment by yanokwa** [comment]
User metadata in ODK Collect can be locked down using the admin mode. Have you tried that?

**Comment by sadiqkhoja** [comment]
yes I've tried admin features of ODK Collect but with that I would need to do the configuration on every data collector's phone, which in my case not possible as they are dispersed across the country
